src/otx/algo/common/layers/transformer_layers.py

```diff
@@ -6,10 +6,14 @@
 from __future__ import annotations
```

Comment:
This file has been moved from rtdert and 3D object detection module. Reviewing it is optional.